### PR TITLE
TFAC-866 Modal for Selling Yahoo

### PR DESCRIPTION
**Pull request by jedtan**

### CODE REVIEW

**Review by kmjennison — CHANGES_REQUESTED** [review]

File: src/components/YahooSellNotification.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+}) => {
+  const [open, setOpen] = useState(true)
+  const classes = useStyles()
+  const onLearnMoreClick = () => {
+    onLearnMore()
+    setOpen(false)
+  }
+  const onNoThanksClick = () => {
+    onNoThanks()
+    setOpen(false)
```
[the remark this answers is not in the document]
Comment (jedtan) [reply src/components/YahooSellNotification.js:59]: Sounds good.

File: .storybook/main.js [hunk 2]
```diff
@@ -1,8 +1,50 @@
+const path = require('path');
+const dotenv = require('dotenv');
+
+// ----------------------------------------
+// Storybook custom env var parsing
+
+const previewDotenv = dotenv.config({
+	path: path.resolve(__dirname, '../.env.preview.info'),
```
Comment (jedtan) [inline .storybook/main.js:8]: Currently the only env vars being complained about are in `.env.preview.info`, so just gonna go with that for now.

Comment (kmjennison) [reply .storybook/main.js:8]: Sounds good.

File: src/components/YahooSellNotification.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  learnButton: {
+    height: '30px',
+    fontWeight: '900',
+    fontFamily: 'Poppins',
+  },
+  noThanksButton: {
+    marginLeft: 'auto',
+    borderRadius: '15px',
+    height: '30px',
+    fontWeight: '900',
```
Comment (kmjennison) [inline src/components/YahooSellNotification.js:21]: This and the "Learn more" button should be font-weight 500:
https://www.figma.com/file/3eY3CAnOdBiQ7eFOP8rnp9/Tab-Legacy%3A-Yahoo-Partnership?node-id=146%3A13055

File: src/components/YahooSellNotification.js [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    fontFamily: 'Poppins',
+  },
+  switchToYahooButton: {
+    background: '#29BEBA',
+    borderRadius: '15px',
+    height: '30px',
+    fontWeight: '900',
+    fontFamily: 'Poppins',
+    marginLeft: theme.spacing(1),
+  },
```
Comment (kmjennison) [inline src/components/YahooSellNotification.js:31]: The "switch to yahoo" button text should be white:
https://6102cd7112cc720039c0b2da-zlxphgosuc.chromatic.com/?path=/story/components-yahoosellnotification--basic&globals=backgrounds.grid:false

**Review by kmjennison — CHANGES_REQUESTED** [review]

**Review by kmjennison — APPROVED** [review]
LGTM! 🦊 Nice work.

**Pull request merged**